server/views/pages/approve/keyDates.njk

```diff
@@ -0,0 +1,45 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% macro keyDates(licence) %}
+
+    {% set dateRows = [
+            {
+                key: { text: "Release date" },
+                value: { text: licence.actualReleaseDate | datetimeToDateShort }
```

Comment:
I think sometimes actualReleaseDate is null and if it is we display conditionalReleaseDate. It might be worth checking if we should do that here